### PR TITLE
docs: update documentation for new v3.0

### PR DESCRIPTION
**Pull request by ghiscoding**
Hi there, I wanted to contribute back for the great work you've done on your new 3.0 release, so I've done a few search & replace (manually) with the info you've put in #316 

There's a few notes though
- I left a TODO in 1 of the settings Russian doc for the `disableWeekdays` option because I think the new option is the inverse of previous option `settings.visibility.weekend`?
- the following options needs to be reviewed/updated or were not found in the docs because they might be new
  - locale
  - enableMonthChangeOnDayClick
  - selectedWeekends
  - timeMinHour
  - timeMaxHour
  - timeMinMinute
  - timeMaxMinute
  - onClickWeekDay
  - onClickTitle
  - onCreateDateRangeTooltip
  - labels
  - layouts
  - styles
  - setContext() new function
  
Another side note, for any potential URLs linking section names, they will need to be updated, for example a link to `## settings.visibility.positionToInput` will change from 
- from: https://vanilla-calendar.pro/docs/reference/additionally/settings#visibility-positiontoinput
- to: https://vanilla-calendar.pro/docs/reference/additionally#positiontoinput  (the url might be wrong here, just displaying as info)

Hope that helps, cheers :)

### CODE REVIEW

**Comment by ghiscoding** [comment]
side note, I saw you added a Ko-Fi link and I tried to send you one but I think there's no account for donation yet, mine is linked to PayPal. Maybe we have to wait couple days?

**Comment by uvarov-frontend** [comment]
@ghiscoding Unbelievable, thank you very much! I am now updating the site and your help is very useful.

I created a page on ko-fi, but I haven’t linked a bank account yet because my Russian account is not suitable for this. 😀 I will need to open a bank account in another country, I’m already doing it.

File: docs/en/reference/additionally/actions.mdx [hunk 1]
```diff
@@ -173,90 +159,82 @@ You can get the following parameters:
...
 
-## actions.getDays()
+## onCreateDateEls()
 
 `Type: Function`
 
 `Default: null`
 
-`Options: getDays(day, date, HTMLElement, HTMLButtonElement, self) => void | null`
+`Options: onCreateDateEls(day, date, mainElement, HTMLButtonElement, self) => void | null`
```
Comment (ghiscoding) [inline docs/en/reference/additionally/actions.mdx:168]: hmm it looks like I missed the `HTMLButtonElement`, not sure what the new name should be. Same problem with line 172 just below

File: docs/en/reference/additionally/actions.mdx [hunk 2]
```diff
@@ -173,90 +159,82 @@ You can get the following parameters:
...
 
-## actions.getYears()
+## onCreateYearEls()
 
 `Type: Function`
 
 `Default: null`
 
-`Options: getYears(year: number, HTMLElement: HTMLElement, self: IVanillaCalendar) => void | null`
+`Options: onCreateYearEls(year: number, mainElement: HTMLElement, self: IVanillaCalendar) => void | null`
```
Comment (ghiscoding) [inline docs/en/reference/additionally/actions.mdx:212]: I missed a type here where `IVanillaCalendar` should be using the new `Calendar`

File: docs/ru/reference/additionally/actions.mdx [hunk 3]
```diff
@@ -175,90 +161,82 @@ new VanillaCalendar('#calendar', {
...
 
-## actions.getMonths()
+## onCreateMonthEls()
 
 `Type: Function`
 
 `Default: null`
 
-`Options: getMonths(month: number, HTMLElement: HTMLElement, self: IVanillaCalendar) => void | null`
+`Options: onCreateMonthEls(month: number, mainElement: HTMLElement, self: IVanillaCalendar) => void | null`
```
Comment (ghiscoding) [inline docs/ru/reference/additionally/actions.mdx:193]: same as above, replace `IVanillaCalendar` with `Calendar`

File: docs/en/reference/additionally/actions.mdx (hunk 1 above)
Comment (ghiscoding) [reply docs/en/reference/additionally/actions.mdx:168]: actually these onX actions internal code seem to only accept 2 arguments `(self, cal)`, so all these new action docs might need to be updated with the 2 args. I'll leave that up to you to verify. :)

File: docs/ru/reference/additionally/actions.mdx [hunk 4]
```diff
@@ -175,90 +161,82 @@ new VanillaCalendar('#calendar', {
...
 
-## actions.getDays()
+## onCreateDateEls()
 
 `Type: Function`
 
 `Default: null`
 
-`Options: getDays(day, date, HTMLElement, HTMLButtonElement, self) => void | null`
+`Options: onCreateDateEls(day, date, mainElement, HTMLButtonElement, self) => void | null`
```
[the remark this answers is not in the document]
Comment (uvarov-frontend) [reply docs/ru/reference/additionally/actions.mdx:170]: Look at this line and this file as a whole: https://github.com/uvarov-frontend/vanilla-calendar-pro/blob/f7b98a9fd4ad769fa9f86dd7fa42e2b4606f4976/package/src/options.ts#L93

This file contains all types and default values ​​for all options.

File: docs/en/reference/additionally/actions.mdx (hunk 1 above)
Comment (uvarov-frontend) [reply docs/en/reference/additionally/actions.mdx:168]: Okay, don't worry, I'll figure it out, thanks for your help)

**Comment by uvarov-frontend** [comment]
@ghiscoding I finally finished with the new documents, it was very boring. I wrote new documents in Russian and then translated them through AI, please let me know if somewhere there is an inaccurate or erroneous translation. Thanks for your help! 

You can also create new documentation pages if something is not in the documentation. Just create an `mdx` file in the appropriate section and it will appear on the site.

**Comment by ghiscoding** [comment]
great to hear it was helpful even if it was boring to review 😉

**Pull request merged**